ansible/roles/test/files/tools/loganalyzer/loganalyzer.py

```diff
@@ -105,8 +106,9 @@ def is_filename_stdin(self, file_name):
 
     #---------------------------------------------------------------------
 
-    def is_filename_sairedis_rec(self, file_name):
-        return "sairedis.rec" in file_name
+    def require_marker_check(self, file_name):
```

Comment:
Please add some documentation here comments, describing why marker checks are not needed for some files.

Reply:
Added some explanation for this function.